src/api/constantDefaults.ts

```diff
@@ -1,2 +1,2 @@
 export const GlobalPath = '~/.pnpm'
-export const GlobalStorePath = GlobalPath + '/.store'
+export const GlobalStorePath = '~/.store'
```

Comment:
hmm, I think usual custom conventions are more like `GLOBAL_PATH = '~/.pnpm'` but thats just a preference for me :)